cdp_scrapers/legistar_utils.py

```diff
@@ -71,13 +72,12 @@
 LEGISTAR_FILE_EXT_ID = "MatterAttachmentId"
 LEGISTAR_FILE_NAME = "MatterAttachmentName"
 LEGISTAR_FILE_URI = "MatterAttachmentHyperlink"
-LEGISTAR_MATTER_EXT_ID = "EventItemMatterId"
-LEGISTAR_MATTER_TITLE = "EventItemMatterFile"
-LEGISTAR_MATTER_NAME = "EventItemMatterName"
-LEGISTAR_MATTER_TYPE = "EventItemMatterType"
-LEGISTAR_MATTER_STATUS = "EventItemMatterStatus"
+LEGISTAR_MATTER_EXT_ID = "MatterId"
+LEGISTAR_MATTER_TITLE = "MatterFile"
+LEGISTAR_MATTER_NAME = "MatterName"
+LEGISTAR_MATTER_TYPE = "MatterTypeName"
+LEGISTAR_MATTER_STATUS = "MatterStatusName"
 LEGISTAR_MATTER_SPONSORS = "MatterSponsorInfo"
-LEGISTAR_SPONSOR_PERSON = "SponsorPersonInfo"
```

Comment:
Dropped this?

Reply:
Yes. So before, we never queried the full Legistar Matter. We just used bits of Legistar EventItem like EventItemMatterName to create ingestion model `Matter`.
Now that there is need to query for a Legistar Matter, I implemented `get_legistar_matter()` so I can reuse it from both the main `get_legistar_events_for_timespan()` and this new `get_updated_model()`...

So, before in `get_legistar_events_for_timespan()`

```
legistar_event_item = {
    ...
    "SponsorPersonInfo" : [
        {
            "PersonFullName" : ...
        },
        ...
    ],
    ...
}
```

Now,

```
legistar_event_item = {
    ...
    "EventItemMatterInfo" : {
        "MatterFile" : ...,
        "MatterSponsorInfo" : [
            {
                "PersonFullName" : ...
            },
            ...
        ],
        ...
    },
    ...
}
```